README.md

```diff
@@ -98,13 +98,7 @@ The following attributes are available for all models of `viamrtsp` discovery se
 
 ### Next steps
 
-Use the `DiscoverResources` API to retrieve a list of cameras discovered by the service and their configuration. Some cameras will output multiple channels, so review the `rtsp_address` of the cameras to determine which camera streams you wish to add.
```

Comment:
I liked including the API name here so it was clear to users what they should be calling on the DiscoveryService, especially if we end up adding more functions to the service down the line. 

This `next steps` section is still under the discovery service heading so it should be implied that the function is part of `viam:viamrtsp:onvif` `discovery`, but if you want to be explicit then that is fine with me

Reply:
Added back!